### PR TITLE
Add incident report for fw0 drive crash

### PR DESCRIPTION
**Pull request by Keeeeeeeen**
Adds an incident report for fw0 drive crash that occured on the 7th/8th of September

### CODE REVIEW

File: docs/incidents/fw0-20240809.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Just after midnight, Ash returns to set up the DHCP server, and then fix a misconfiguration in it in order to disable IPv6. Then, at around 1pm, Ash installs and configures Squid, before realising that IRN has removed IP Based Authentication, and, as such, the Squid proxy is no longer needed. Ash then configures the firewall script to run on boot, completing all remaining critical recorvery tasks
+
+===INCIDENT END===
+
+## Highlighted Future Action Points
+
+1.  Investigate why monitoring didn't alert us earlier to a drive crash
+2.  Ensure we have up to date backups of our config files
+    -   Stratford stores these
```
Comment (markspolakovs) [inline docs/incidents/fw0-20240809.md]:
```suggestion
    -   Stratford and Moyles store these
```

**Review by markspolakovs — COMMENTED** [review]
LGTM for now, though we may want to do a proper post-incident review and root cause analysis (beyond the scope of this PR besides updating it with the outcome).

**Comment by Keeeeeeeen** [comment]
I assumed a proper post incident review would probably come first comp meeting back, at which point we can update it, I just mostly wanted to get the events of the day down while they were fresh in my mind for historical purposes

**Comment by hitime1234** [comment]
Other than that suggestion LGTM

**Comment by JamboPE** [comment]
Looks good

**Comment by archessmn** [comment]
Looks good :rocket:

**Comment by alyxbb** [comment]
LGTM

File: docs/incidents/fw0-20240809.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Many things had came back online now, but by 6pm, MyRadio was still taking incredibly long to load. By 6:20, Marks diagnoses the issue as Docker DNS relying on fw0, which hadn't been configured correctly to handle this post-reinstall. Marks then installs Unbound, and MyRadio begins to work as normal once again. At this point, the Incident is delared ended.
+
+### Monday, September 9th, Final Critical Recovery Tasks
+
+Just after midnight, Ash returns to set up the DHCP server, and then fix a misconfiguration in it in order to disable IPv6. Then, at around 1pm, Ash installs and configures Squid, before realising that IRN has removed IP Based Authentication, and, as such, the Squid proxy is no longer needed. Ash then configures the firewall script to run on boot, completing all remaining critical recorvery tasks
+
+===INCIDENT END===
```
Comment (ashhhleyyy) [inline docs/incidents/fw0-20240809.md]:
```suggestion
**===INCIDENT END===**
```

**Review by ashhhleyyy — DISMISSED** [review]

**Review by JamboPE — APPROVED** [review]

**Review by JamboPE — APPROVED** [review]

**Pull request merged**